model.py

```diff
@@ -129,11 +132,11 @@ def __update_statement(self):
         ).where(ServerConfigModel.server_id == self.server_id)
         return stmt
 
-    async def sync_to_db(self, async_engine_generator: Callable[[bool], contextlib.AbstractAsyncContextManager[AsyncConnection]]):
+    async def sync_to_db(self, bot: WordChainBot):
```

Comment:
I prefer the old variant, but I can live with both. The old one avoided the cyclic dependency, while this one makes clear what the `bool` is needed for.

Reply:
Remnant from the time when the `db_connection()` function was in the erstwhile `utils.py` file, but I believe `WordChainBot` is somewhat easier to read than the entire `contextlib....`.